packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/TransactionsCompletedRow.tsx

```diff
@@ -1,37 +1,43 @@
 import React from 'react';
-import { useActiveWalletAccount } from '@deriv/api';
-import { WalletListCardBadge } from '../../../../../../components';
 import { WalletText } from '../../../../../../components/Base';
-import { WalletCurrencyCard } from '../../../../../../components/WalletCurrencyCard';
 import { THooks } from '../../../../../../types';
+import { TransactionsCompletedRowAccountDetails } from './components/TransactionsCompletedRowAccountDetails';
+import { TransactionsCompletedRowTransferAccountDetails } from './components/TransactionsCompletedRowTransferAccountDetails';
 import './TransactionsCompletedRow.scss';
 
 type TProps = {
     transaction: THooks.Transactions;
+    wallet: THooks.ActiveWalletAccountVerbose;
 };
 
-const TransactionsCompletedRow: React.FC<TProps> = ({ transaction }) => {
-    const { data: wallet } = useActiveWalletAccount();
```

Comment:
Shouldn't have done fetching in the row component earlier as it's literally race condition